### PR TITLE
Feature/create minows

### LINKED ISSUE #2: Clone minoch implementation into minows
- Create a new directory with the files
- add a README.md to the new directory with information as to how we want to implement it
- make sure all tests in the new directory still run

### PR DESCRIPTION
**Pull request by XioZ**
Closing #2

### CODE REVIEW

File: mino/minows/architecture.png
Comment (ineiti) [inline mino/minows/architecture.png:1]: Please change the IP address and the domain-name `p2p-[12].dela-1.c4dt.org`

Comment (XioZ) [reply mino/minows/architecture.png:1]: Updated the architecture diagram & texts accordingly

Comment (ineiti) [inline mino/minows/architecture.png:1]: The entry for the public address at 443 is only WSS, not WS.

**Review by ineiti — DISMISSED** [review]

File: mino/minows/architecture.png
Comment (XioZ) [reply mino/minows/architecture.png:1]: Added port 8080

Comment (ineiti) [reply mino/minows/architecture.png:1]: WS uses http, so it uses normally port 80. I propose to just remove the "WS" from the traefik.

**Pull request merged**